classes/webservice/WebserviceRequest.php

```diff
@@ -1422,7 +1422,7 @@ protected function executeEntityDelete()
                 }
 
                 /* @var ObjectModel $object */
-                if (isset($this->resourceConfiguration['objectMethods']) && isset($this->resourceConfiguration['objectMethods']['delete'])) {
+                if (isset($this->resourceConfiguration['objectMethods'], $this->resourceConfiguration['objectMethods']['delete'])) {
```

Comment:
you can simplify with `isset($this->resourceConfiguration['objectMethods']['delete'])`